### ISSUE TITLE
Import MySQL data, ID error

### ISSUE THREAD

**Issue by shuzuo** [jprante/elasticsearch-jdbc#894]
data
type                        _id                                    id
enterpriseMaterial 806450891548564141 1 806450891548564100 
enterpriseMaterial 806450891548564146 1 806450891548564100 

sql
SELECT `ID` AS `_id`,`ID` .........

There are a lot of similar data

**Comment by shuzuo** [comment]
@jprante help mi!

**Comment by shuzuo** [comment]
@jprante , it's elasticsearch-head plugin display problem

**Comment by cch123** [comment]
yes, because javascript cannot support int64 data and will lose precision if your number is long than 16digit, but the data in es is correct

**Comment by jprante** [comment]
You can try to convert to string with SQL `CAST` or `CONVERT` operator http://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#function_cast
